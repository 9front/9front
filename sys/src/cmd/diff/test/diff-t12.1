# Test HMAC:

PROG=		hmactest
SRCS=		hash.c hmactest.c
.PATH:		${.CURDIR}/../../
NOMAN=
CFLAGS+=	-I${.CURDIR}/../../ -Wall
DEBUG=		-g

.include <bsd.prog.mk>