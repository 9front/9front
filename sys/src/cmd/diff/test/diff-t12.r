#	$OpenBSD: t12.2,v 1.1 2003/07/21 20:16:21 otto Exp $

# Test HMAC:

PROG=		hmactest
SRCS=		hash.c hmactest.c
.PATH:		${.CURDIR}/../../
NOMAN=
CFLAGS+=	-I${.CURDIR}/../../ -Wall
DEBUG=		-g

.include <bsd.prog.mk>