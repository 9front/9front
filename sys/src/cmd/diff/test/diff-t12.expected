--- diff-t12.1
+++ diff-t12.2
@@ -1,3 +1,5 @@
+#	$OpenBSD: t12.2,v 1.1 2003/07/21 20:16:21 otto Exp $
+
 # Test HMAC:
 
 PROG=		hmactest
